### DISCUSSION TITLE
Could not find org.apache.cordova:framework:7.0.0

### DISCUSSION THREAD

**Discussion by joshstrange** [apache/cordova#414] [General]
# Bug Report

You can no longer install any version of the cordova framework <10

## Problem

JFrog has [broken it's promise](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) to keep jcenter up "indefinitely".

> UPDATE 4/27/2021: We listened to the community and will keep JCenter as a read-only repository indefinitely. Our customers and the community can continue to rely on JCenter as a reliable mirror for Java packages.

They are currently [just redirecting to maven central](https://status.gradle.com/incidents/7x4wqd7zv715) which does not have anything before version 10

### What is expected to happen?

Older cordova builds continue to work

### What does actually happen?

It's impossible to build older corodva apps and it's currently impossible to even build 7.0.0 (and I assume anything <10) locally as it depends on things only in jcenter

## Information

This issue is not for apache/cordova-android to do anything, but rather as a way for the community to find information when they search for this issue in their builds. Also I'd love if anyone has any workaround (other than upgrading, we are investigating that now). I have the .pom and .aar files for 7.0.0 but I'm unable to get them to work (by adding them to the source).

### Command or Code

N/A

### Environment, Platform, Device

N/A

### Version information

N/A

## Checklist

- [x] I searched for existing GitHub issues
- [x] I updated all Cordova tooling to most recent version
- [x] I included all the necessary information above

See same discussion on [ionic-team/capacitor](https://github.com/ionic-team/capacitor/discussions/6672)

**Comment by breautek** [comment]
> This issue is not for apache/cordova-android to do anything, but rather as a way for the community to find information when they search for this issue in their builds. 

I agree... and for this reason I think a better place for this is inside our github discussions, I'll move this there.

**Reply by joshstrange** [reply]
My apologizes, thank you

**Comment by jcesarmobile** [comment]
The incident has been resolved.

But I would encourage everybody using cordova-android as a gradle dependency to move to version 10.0.0 or newer as those version is available on maven central.
Or if using cordova-android in their apps, also move to 10.0.0 or newer as it removed jcenter dependencies it was using or updated them to use a maven central version. (Even better if they use 12.0.0 or newer as that's going to be required to submit apps to Google Play starting in August)

**Comment by sagadav** [comment]
I ran into this problem on a project with quasar js, and just changing the version didn't help me. It turned out that I use the second version of capacitor (in src-capacitor), so

```
cd src-capacitor
npm i -D @capacitor/cli@latest-5
npx cap migrate
```

Maybe this will help someone not to waste a lot of time.
